src/hooks/query/auth/useLogin.js

```diff
@@ -14,7 +15,8 @@ export const useLogin = () => {
   const { mutate, isLoading } = useMutation({
     mutationKey: [queryKeys.login],
     mutationFn: login,
-    onError: () => {
+    onError: (error) => {
+      loginError(error)
       // TODO: Vitalii task: add error notification
```

Comment:
1. if you did it, remove TODO comment
2. looks like you don't need any wrapp function, can be -> onError: loginError
3. Could you create helper function in helpers folder "handleRequestError" then we will be able to show error everywhere where it's needed.

Reply:
I create handleRequestNitification.js with succsess and error notification. They reusable.